### PR TITLE
 Instructor: question edit: Use different color for inner panel titles #2868 

### LINKED ISSUE #2868: Instructor: question edit: Use different color for inner panel titles
Using a different color set them apart from other bold text in the panel. Use the darker tone of the panel background color?

image

### PR DESCRIPTION
**Pull request by chowyb**
Fixes #2868

### CODE REVIEW

**Comment by chowyb** [comment]
Ready for review

**Comment by JosephineKwa** [comment]
Hey @chowyb could you post screenshots of the change

**Comment by chowyb** [comment]
@JosephineKwa 

questionedit

The color of the text in red boxes has been changed from the previous color (black), following the previous discussion on this issue in #3153 and #3194.

File: src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 <div class="col-sm-12 padding-15px margin-bottom-15px background-color-light-green">
     <div class="col-sm-12 padding-0 margin-bottom-7px">
-        <b>Feedback Path</b> (Who is giving feedback about whom?)
+        <b class="feedbackPathTitle">Feedback Path</b> (Who is giving feedback about whom?)
```
Comment (JosephineKwa) [inline src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag]: Class names should be lowercase and delimited by hyphens based on our naming conventions, do edit this :)

**Comment by chowyb** [comment]
@JosephineKwa Ready for review :)

File: src/main/webapp/stylesheets/teammatesCommon.css [hunk 2]
```diff
@@ -566,6 +566,11 @@ Apply to thead to make headers not bold.
     width: 12px;
 }
     
+.feedback-path-title,
+.visibility-title {
+	color: #2aabd2;
```
Comment (JosephineKwa) [inline src/main/webapp/stylesheets/teammatesCommon.css]: The indentation should be 4 spaces

**Comment by chowyb** [comment]
@JosephineKwa fixed, ready for review :)

**Comment by wkurniawan07** [comment]
@damithc this one has been pending your review as well.

**Comment by damithc** [comment]
Sorry, missed it earlier.

**Pull request merged**